javascript/core/recipes-i/quick-way-to-format-integers-with-a-fixed-number-of-digits.md

```diff
@@ -34,8 +34,9 @@ links:
 
 
 aspects:
-  - introduction
   - workout
+  - obscura
+  - deep
```

Comment:
I would leave `introduction`, and remove `deep`. It's fine with `obscura` though.